src/vs/workbench/api/common/extHostCommands.ts

```diff
@@ -46,6 +46,7 @@ export class ExtHostCommands implements ExtHostCommandsShape {
 
 	private readonly _commands = new Map<string, CommandHandler>();
 	private readonly _apiCommands = new Map<string, ApiCommand>();
+	private readonly _telemetry: MainThreadTelemetryShape;
```

Comment:
This introduces integration test failures because the `MainThreadTelemetryShape` is accessible to extension via the "not so private at runtime" `_telemetry`-property. For this case, it's best to use a native private, similar to the `#proxy` member below